packetbeat/tests/system/test_0062_cassandra.py

```diff
@@ -381,7 +399,9 @@ def test_compressed_frame(self):
             cassandra_compressor="snappy",
         )
 
-        self.run_packetbeat(pcap="cassandra/v4/cassandra_compressed.pcap", debug_selectors=["*"])
+        self.run_packetbeat(
+            pcap="cassandra/v4/cassandra_compressed.pcap",
+            debug_selectors=["*"])
         objs = self.read_output()
 
         o = objs[0]
```

Comment:
None of this changes to packetbeat/tests/system/* files seems related to this particular PR.

If you really meant to submit this code formatting changes, please do so in a separate PR